src/Fixer/PhpTag/BlankLineAfterOpeningTagFixer.php

```diff
@@ -76,17 +69,18 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
             return;
         }
 
-        $token = $tokens[0];
+        $openTagIndex = $tokens[0]->isGivenKind(T_INLINE_HTML) ? 1 : 0;
+        $token = $tokens[$openTagIndex];
 
         if (!str_contains($token->getContent(), "\n")) {
-            $tokens[0] = new Token([$token->getId(), rtrim($token->getContent()).$lineEnding]);
+            $tokens[$openTagIndex] = new Token([$token->getId(), rtrim($token->getContent()).$lineEnding]);
         }
 
-        if (!str_contains($tokens[1]->getContent(), "\n")) {
-            if ($tokens[1]->isWhitespace()) {
-                $tokens[1] = new Token([T_WHITESPACE, $lineEnding.$tokens[1]->getContent()]);
+        if (isset($tokens[$openTagIndex + 1]) && !str_contains($tokens[$openTagIndex + 1]->getContent(), "\n")) {
+            if ($tokens[$openTagIndex + 1]->isWhitespace()) {
+                $tokens[$openTagIndex + 1] = new Token([T_WHITESPACE, $lineEnding.$tokens[$openTagIndex + 1]->getContent()]);
             } else {
-                $tokens->insertAt(1, new Token([T_WHITESPACE, $lineEnding]));
+                $tokens->insertAt($openTagIndex + 1, new Token([T_WHITESPACE, $lineEnding]));
```

Comment:
Maybe we can introduce a variable like e.g. `$newLineIndex` to avoid repeating `$openTagIndex + 1`.

Reply:
why not, landed